src/main/tut/2.4-representable-functors.md

```diff
@@ -0,0 +1,208 @@
+```Haskell
+C(a, x) -> C(a, y)
+```
+```scala
+`C(A, X)` => `C(A, Y)`
+```
+................
+```Haskell
+f ∘ h :: a -> y
+```
+```scala
+val `f ∘ h`: A => Y
```

Comment:
This scala code is not the same as the haskell above:
in the haskell snippet `a`  and `y` are universally quantified, whereas in the scala code they are concrete types called `A` and `Y`.
It should be ````def `f ∘ h`[A, Y] : A =>  Y```